oaknational/Oak-Web-Application
# chore: remove z-index from component LESQ-446

## Description

- Removes z-index from component

## Issue(s)

Fixes #LESQ-446

## How to test

1. Go to https://deploy-preview-2074--oak-web-application.netlify.thenational.academy
3. Click on school picker dropdown menu
4. You should see renders above the checkbox

## Screenshots

How it used to look (delete if n/a):
![Screenshot 2023-11-03 at 08 09 21](https://github.com/oaknational/Oak-Web-Application/assets/91190841/68ba8abf-008b-4344-8223-7e7210c1f6d8)

How it should now look:
![Screenshot 2023-11-03 at 08 07 30](https://github.com/oaknational/Oak-Web-Application/assets/91190841/1f7cc3d3-a688-46fc-bc7f-2ee771ac6cf6)


## Checklist
